wger/core/views/user.py

```diff
@@ -307,6 +314,66 @@ def preferences(request):
         return render(request, 'user/preferences.html', template_data)
 
 
+@login_required
+def add_fitbit(request, code=None):
+    '''
+    Gets data from fitbit upon the user authorizing Wger to access their data
+    '''
+    template_data = {}
+    client_id = '2283MF'
+    client_secret = 'c8ebd0a368cf7f419102198633966039'
```

Comment:
Meant to put it in the settings file but didn't have the time. I shall be doing that shortly